docs/.eslintrc.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
       }
     },
     {
-      files: ['SvgFrameworkIcons.vue', 'Navbar.vue'],
+      files: ['SvgFrameworkIcons.vue', 'Navbar.vue', '.vuepress/config.js'],
```

Comment:
It would be best to create a new file containing 3rd-party scripts and include that file here, rather than excluding the entire config file from the rule. WDYT?